### ISSUE TITLE
Tooltip does not close when exiting the triggering item - violation of 1.4.13?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#1193]
I have to evaluate an application according to WCAG where large tooltips hiding the content are automatically displayed when the links with the tab key get the focus. The tooltips can be closed with ESC as long as the focus is on the link. However, when the focus is moved further with the keyboard, the tooltips remain open and cannot be closed with ESC. This means that to close the tooltip, the focus must be on the link that triggered it. Is this a violation of SC 1.4.13?

**Comment by jake-abma** [comment]
Interesting scenario, I would say NO, as the SC says:

"receiving **and then** removing keyboard focus" ... "additional content to become visible and then hidden"

Receiving / visible = YES
Removing / hidden = NO

**Comment by JAWS-test** [comment]
@jake-abma It makes sense. The question is whether this is a gap in SC 1.4.13, since it is of course even more annoying for keyboard users if automatically displayed content cannot be hidden at all (as long as the problem does not occur for mouse users)

**Comment by guyhickling** [comment]
I consider this possibility is covered by SC2.1.1 Keyboard, since it says: "All functionality of the content is operable through a keyboard interface". Part of the functionality of your content is that mouse users can hover to show the tooltip, then simply remove that trigger (the mouse hover) to hide the tooltip. Therefore that same functionality of hiding by the user simply removing the trigger must be available from the keyboard by removing the focus (by tabbing to the next or previous component). I would (and do, though it is rare to see this particular case) raise it as such in an audit. 

NB: It is also very poor usability and therefore accessibility for keyboard users if they have to press Esc on every link and button just to stop the page filling up with unwanted clutter.

**Comment by detlevhfischer** [comment]
@guyhickling I do not think this gap is properly closed by 2.1.1 Keyboard. I cannot see a requirement for a strict functional equivalence between pointer and keyboard operability, Arguably, the situation of the lingering tooltip might still be considered operable (tabbing back and then ESC) even though this is inconvenient and certainly a bad practice. It would be difficult to include the lingering tooltip scenario in an update to the normative text of 1.4.13 - there are valid scenarios where moving the focus does not immediately close the tooltip (e.g. when it contains focusable content).

**Comment by JAWS-test** [comment]
> there are valid scenarios where moving the focus does not immediately close the tooltip (e.g. when it contains focusable content).

Exactly, but after leaving the triggering element and the interactive elements in the tooltip, the tooltip should close automatically. I don't consider this a violation of 2.1.1 and 1.4.13 either. The question would be whether this gap should be closed with a new SC or whether this might not be relevant enough (because it happens too rarely).

**Comment by mraccess77** [comment]
The way the SC is worded the SC may not even apply if the additional content stays up if triggered on focus/hover -- but if either focus or mouse movement causes the the additional content to disappear then the SC is applicable.
